admin/views/restart/restart-overview.ejs

```diff
@@ -0,0 +1,54 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <header class="page-header">
+            <h1 class="heading-xlarge">Restarts</h1>
+        </header>
+        <br />
+        <span class="font-medium">
+            If a pupil was significantly interrupted while taking the check,
+            use this page to notify the STA that they will restart the check.
+            The pupil will take a different check form from their previous attempt
+            and the last completed attempt will be reported.
+        </span>
+    </div>
+    <div class="column-one-third add-title-margin" tabindex="0" aria-label="sidebar">
+        <aside role="complementary" class="support-column">
+            <h3 class="heading-medium" tabindex="0">Related</h3>
```

Comment:
Why add tabindex here?  GDS pattern is here: https://govuk-elements.herokuapp.com/typography/#typography-headings

Reply:
@jon-shipley yes they aren't needed, removed